deps.sh

```diff
@@ -20,4 +20,4 @@ pip3 install --upgrade pip
 # It's nice to unit test, integration test, and run the CLI in
 # a development pyenv.
 #
-pip3 install -r requirements.txt -e '.[unittest,itest,cli,typecheck]'
+pip3 install -r requirements.txt -e '.[unittest,itest,cli,typecheck,docs]'
```

Comment:
With this extra, you can run 'make docs' in the docs/ directory and generate things locally.